### DISCUSSION TITLE
New Product Alpha Testing

### DISCUSSION THREAD

**Discussion by zolotokrylin** [clearpool-finance/community#16] [Announcements]
We are inviting all Clearpool community members to participate in our Alpha Testing campaign as we launch a new product.
The campaign will be launching very soon, so don't forget to subscribe to this thread. We will post an update here with the rules and terms of participation. Juicy rewards will be distributed to outstanding contributors.

**Comment by zolotokrylin** [comment]
We are launching a new product soon. If anyone would like to participate in smart-contracts bounty hunting, please let me know.

**Reply by g01u** [reply]
Yes sir

**Reply by zolotokrylin** [reply]
@g01u please register here https://tally.so/r/mVj92M

**Reply by kalpshah284** [reply]
Ok sir I registered, Few month ago I submitted bug in clear pool please sir consider it for giving me opportunity of job.Thank you

**Reply by zolotokrylin** [reply]
yes @kalpshah284. I remember you. I just messaged you in Discord in our common chat.
